src/main/sensors/gyroanalyse.c

```diff
@@ -103,7 +110,14 @@ void gyroDataAnalyseInit(uint32_t targetLooptimeUs)
         hanningWindow[i] = (0.5f - 0.5f * cos_approx(2 * M_PIf * i / (FFT_WINDOW_SIZE - 1)));
     }
 
-    dynamicNotchCutoff = (100.0f - gyroConfig()->dyn_notch_width_percent) / 100;
+    dynamicFilterCutoffFactor = (100.0f - gyroConfig()->dyn_filter_width_percent) / 100;
+    dynamicFilterType = gyroConfig()->dyn_filter_type;
+    dynamicFilterThreshold = gyroConfig()->dyn_filter_threshold / 10;
+    if (gyroConfig()->dyn_filter_location == 1) {
+        dynamicFilterIgnore = gyroConfig()->dyn_filter_ignore;
+    } else {
+        dynamicFilterIgnore = gyroConfig()->dyn_filter_ignore / 10;
```

Comment:
Not sure if it's a good idea to make the threshold dependent on the location - would using a larger range / different scaling for the threshold achieve the same in a more consistent manner?

Reply:
This is a bit of a tough one.  
When the FFT input is from un-filtered gyro, i.e. location 1, the FFT bins are all a lot bigger than when input is post-static filters (location 2).   A suitable default for the pre-filter location is about 30, and it turned out that the post-filter default was near enough to 10 times smaller.
Rather than providing separate default values for each, it seemed simpler to allow the same value in the CLI to be a reasonable default regardless of location.  
It may turn out that the two are not simply interchangeable, but short of providing separate defaults altogether, it seemed OK this way.  It is nice because the number chosen is in the same range for the same effect.

Reply:
I see, seems to make sense then, as long as the scaled down value in the PRE filter has the same effect as the original value in the POST filter over the valid input range - if not then I suspect this simplification will result in basically every user using it asking 'why does this break when I switch from POST to PRE?'

Reply:
It probably won't break when they switch, that's what we're hoping; performance should be very similar, though it can't be identical.  Probably they will just try PT1 and end up back with Notch, in which case they retain their original setting, if they had altered it.